.vscode/launch.json

```diff
@@ -26,6 +28,7 @@
 			"request": "launch",
 			"name": "Run Unit Tests",
 			"program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+			"args": ["--timeout", "999999"],
```

Comment:
This probably shouldn't be part of this PR?

Reply:
ill pull it out

Reply:
posterity https://github.com/SpyglassMC/Spyglass/pull/1189